security/pkg/probe/controller_test.go

```diff
@@ -52,24 +52,25 @@ func TestGcpGetServiceIdentity(t *testing.T) {
 		err      string
 		expected string
 	}{
-		"Check success": {
+		// TODO: Add more tests.
```

Comment:
Can you be specific what test cases to add?

Reply:
Done.

Reply:
Need to test the successful case. Didn't add it to this PR since it requires to touch more files. Will do it after 1.1.